addon/components/wizard/container.hbs

```diff
@@ -1,7 +1,9 @@
 <div class={{or this.wizardManager.configOptions.containerClass "wizard-container"}}>
-  {{#if this.wizardManager.initialized}}
-    {{#each this.allSteps as |step|}}
-      <Wizard::StepWrapper @step={{step}} />
-    {{/each}}
-  {{/if}}
+  <div class="wizard-container__scrollable">
```

Comment:
as discussed yesterday, I think we might be able to remove this sub-div and use a dedicated `configOptions.containerClass` from publishr.

If it doesn't work, then we can keep this implem